Point users to the configuration page for overriding update locks (#1736)
balena-os/balena-supervisor: docs/update-locking.md

To do this the device or fleet must have `BALENA_SUPERVISOR_OVERRIDE_LOCK` configuration variable set to "1".

> The easiest way to do this is to use the 'Override the update lock' toggle in the [Fleet or Device Configuration][device-configuration] page. Go to the configuration page of the device you would like to override the lock, then locate the 'Override the update lock' item, click the activate button and then set the toggle to enabled. To get the update locks working again, disable this toggle afterwards.

Reviewer: ```suggestion
The easiest way to do this is to use the 'Override the update lock ...' toggle in the [Fleet or Device Configuration][device-configuration] page. Go to the configuration page of the device or fleet, locate the 'Override the update lock ...' item, click the activate button, and set the toggle to enabled. Disable the toggle afterwards in order to restore the update locks.
```

Slight rewording to shorten the middle sentence with all the steps. Include ellipsis to indicate that there is more text in the UI following `Override the update lock ...`.

Reviewer 2: I agree this reads better.
